File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        String integer = String.valueOf(num);
        String kk = String.valueOf(k);
        for(int i = 0; i < integer.length(); i++){
            if(integer.charAt(i) == kk.charAt(0)){
                return i+1;
            }
        }
        return -1;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.cs
using System;

public class Solution {
    public int solution(int num, int k) {
       string integer = num.ToString();
    string kk = k.ToString();
    
    for (int i = 0; i < integer.Length; i++)
    {
        if (integer[i] == kk[0])
        {
            return i + 1;
        }
    }
    return -1;
    }
}